lib/evm/gas.ex

```diff
@@ -20,7 +20,7 @@ defmodule EVM.Gas do
   @g_high 10  # Amount of gas to pay for operations of the set Whigh.
   @g_extcode 700  # Amount of gas to pay for operations of the set Wextcode.
   @g_balance 400  # Amount of gas to pay for a BALANCE operation.
-  @g_sload 200  # Paid for a SLOAD operation.
+  @g_sload 50  # Paid for a SLOAD operation.
```

Comment:
I think the common tests have to opcode pricing of Metropolis part 1. This also raises the point that gas prices depend on block height (ie, they've changed at forks). Eventually we'll need to build that in.

Reply:
Yeah, I'm debating adding `block_header` directly into EVM (taking it out of Blockchain), since it's used pretty heavily throughout some of the EVM code.